train.py

```diff
@@ -1,12 +1,17 @@
+import csv
 import random
 
 from autowebcompat import network, utils
 
 labels = utils.read_labels()
 
+def update_all_image_names(images):
```

Comment:
No need to define a function for this very simple code, just do the filtering inline